### PR TITLE
fix: make create_instance work with command line

### PR DESCRIPTION
**Pull request by skuruppu**
## Checklist
- [ ] **Tests** pass
- [ ] **Lint** pass: `bundle exec rubocop`
- [x] Please **merge** this PR for me once it is approved.

### CODE REVIEW

**Review by hengfengli — APPROVED** [review]
LGTM. Thanks for the fix.

File: spanner/spanner_samples.rb [hunk 1]
```diff
@@ -2172,6 +2172,9 @@ def run_sample arguments
                           database_id: database_id,
                           start_title: arguments.shift,
                           end_title:   arguments.shift
+  elsif command.eql?("create_instance") && instance_id
+    create_instance project_id:  project_id,
```
[the remark this answers is not in the document]
Comment (skuruppu) [reply spanner/spanner_samples.rb:2176]: I can, but it seems that commands is meant to be a comprehensive list of all commands. Like anything outside of that is not a valid command. For example, `query_data_with_index` is also in there even though it is a special case.

**Review by thiagotnunes — APPROVED** [review]

**Comment by skuruppu** [comment]
@dazuma it seems that the test failures are unrelated to this PR. The "Ruby Newest" and "Ruby Oldest" checks are stuck but I don't think these are Kokoro jobs so I don't know how to trigger them again.

**Comment by hengfengli** [comment]
@dazuma Gentle ping.

**Comment by skuruppu** [comment]
There is a [failure](https://source.cloud.google.com/results/invocations/2c80530f-3a40-43a0-8d93-c0b35888c833/targets/github%2Fruby-docs-samples%2Fspanner/tests) in "Ruby Newest" related to Spanner, which is:

```
Failure/Error:
  write_read_bool_array project_id: @project_id,
                        instance_id: @instance.instance_id,
                        database_id: database.database_id

ArgumentError:
  wrong number of arguments (given 1, expected 2)
./spanner_samples.rb:1506:in `write_read_bool_array'
./spec/spanner_samples_spec.rb:1456:in `block (3 levels) in <top (required)>'
./spec/spanner_samples_spec.rb:128:in `capture'
./spec/spanner_samples_spec.rb:1455:in `block (2 levels) in <top (required)>'
```

But we didn't touch this code in this PR. I stared at the lines specified in the stack trace and I couldn't find an issue. Also the same test passes in [Ruby 2.7](https://source.cloud.google.com/results/invocations/62b04725-27c9-42fb-b531-cb7f98cb9334/targets;collapsed=) which is the only version of Ruby the Spanner tests are actually run against.

@dazuma what is the version used by "Ruby Newest"?

**Review by NivedhaSenthil — APPROVED** [review]

**Pull request closed without merging**